modules/express/src/clientRoutes.ts

```diff
@@ -828,7 +855,7 @@ function createSendParams(req: express.Request) {
   }
 }
 
-function createTSSSendParams(req: express.Request) {
+function createTSSSendParams(req: express.Request, wallet?: Wallet) {
```

Comment:
shouldn't wallet be required? Are there routes that aren't passing it in?

Reply:
its not required for Eddsa since we dont check the multisigTypeVersion, but i can make it required